inst/www/shared/shiny.js

```diff
@@ -3296,6 +3299,14 @@
         sendOutputHiddenStateDebouncer.immediateCall();
     });
 
+    function filterEventsByNamespace(namespace, handler) {
+      return function(e) {
+        if (e.namespace === namespace) {
```

Comment:
No, it doesn't. Sorry, I must have pushed the wrong commit as I was experimenting.